pkg/controller/keystone/keystone_controller.go

```diff
@@ -120,12 +120,26 @@ func (r *ReconcileKeystone) Reconcile(request reconcile.Request) (reconcile.Resu
 		return reconcile.Result{}, nil
 	}
 
+	keystoneService := newKeystoneService(keystone)
+	_, err := controllerutil.CreateOrUpdate(context.Background(), r.client, keystoneService, func() error { return nil })
+
 	keystonePods, err := r.listKeystonePods(keystone.Name)
 	if err != nil {
 		return reconcile.Result{}, fmt.Errorf("failed to list command pods: %v", err)
 	}
 
-	if err := r.ensureCertificatesExist(keystone, keystonePods); err != nil {
+	keystoneServiceList := &core.ServiceList{}
```

Comment:
I would extract it to a separate method moreover I'm thinking that maybe we should watch on changes on that service (I know that in docs it was said that ClusterIP won't change but what about situation when someone will delete svc by accident or not)